chromium_src/third_party/blink/public/common/web_preferences/web_preferences_mojom_traits.h

```diff
@@ -0,0 +1,36 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_THIRD_PARTY_BLINK_PUBLIC_COMMON_WEB_PREFERENCES_WEB_PREFERENCES_MOJOM_TRAITS_H_
+#define BRAVE_CHROMIUM_SRC_THIRD_PARTY_BLINK_PUBLIC_COMMON_WEB_PREFERENCES_WEB_PREFERENCES_MOJOM_TRAITS_H_
+
+#include "third_party/blink/public/common/web_preferences/web_preferences.h"
+#include "third_party/blink/public/mojom/webpreferences/web_preferences.mojom.h"
+
+#define WebPreferences WebPreferences_ChromiumImpl
+
+#include "src/third_party/blink/public/common/web_preferences/web_preferences_mojom_traits.h"
+
+#undef WebPreferences
+
```

Comment:
Should we return `WebPreferences` to the original value here?
@goodov could you also take a look at these defines?

Reply:
yep, this is correct.